docroot/themes/humsci/humsci_basic/templates/components/global-footer.html.twig

```diff
@@ -0,0 +1,51 @@
+{#
+  /**
+  * @file
+  * Global Footer Component.
```

Comment:
can we also say a comment about how we copied this from stanford basic and link to that template?

Reply:
please include this link! https://github.com/SU-SWS/stanford_basic/blob/8.x-4.x/dist/templates/decanter/components/global-footer/global-footer.twig